ci/infra/testrunner/testrunner.py

```diff
@@ -132,6 +132,10 @@ def main():
                         deployed nodes in your platform")
     cmd_bootstrap.add_argument("-k", "--kubernetes-version", help="kubernetes version",
                                dest="kubernetes_version", default=None)
+    cmd_bootstrap.add_argument("-c", "--cloud-provider",
```

Comment:
Do we need to have the cloud provider here? the test runner already knows the platform by means of the `--platform` argument. Does it make sense that `--platform` and `--cloud-provider` has different values? I think not. Therefore, I would make this option to store a boolean.